tests/test.spec.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
```

Comment:
I think you can have a try with [jest-electron](https://github.com/hustcc/jest-electron), I think will be better then mock the component.

It's not necessary for this pr.